js/app/components/chat/chat-events.ts

```diff
@@ -0,0 +1,13 @@
+import { EventEmitter } from "events";
+import { MessageViewHydrated } from "features/player/playerSlice";
+
+// Create a global event emitter for cross-platform event handling
+const globalEventEmitter = new EventEmitter();
```

Comment:
Could this file be handled as a Redux action instead? And new state that shows the reply prompt?